cmake/modules/RootNewMacros.cmake

```diff
@@ -211,6 +211,20 @@ function(ROOT_GET_LIBRARY_OUTPUT_DIR result)
   SET(${result} "${library_output_dir}" PARENT_SCOPE)
 endfunction(ROOT_GET_LIBRARY_OUTPUT_DIR)
 
+#---------------------------------------------------------------------------------------------------
+#---ROOT_GET_INSTALL_DIR( result_var )
+# Returns the path to the shared libraries installation directory. On Windows the pcms and rootmap
+# files must go in the $ROOTSYS/bin folder.
```

Comment:
Why do they need to go into the `bin` folder? I think the better way is to create variables for them, like `CMAKE_INSTALL_PCMDIR`, `CMAKE_INSTALL_MODULEDIR`, etc, then set each new variable to either `CMAKE_INSTALL_BINDIR` or `CMAKE_INSTALL_LIBDIR` depending on the system.

Reply:
On Windows, they have to be next to the DLLs (and they are created in the bin directory). This is to solve the issue at install time (creating variables for each of them is over-complicating it)

Reply:
Because the `.lib` and `.exp` files must go in the `lib` directory